src/auth/Cargo.toml

```diff
@@ -22,7 +22,7 @@ tracing = "0.1.21"
 tracing-futures = "0.2.4"
 x509-parser = "0.9.1"
 
-fluvio-controlplane-metadata = { version = "0.7.0", path = "../controlplane-metadata" }
+fluvio-controlplane-metadata = { version = "0.8.0", path = "../controlplane-metadata" }
```

Comment:
reason for updating to `0.7.0`?

Reply:
Yes, we changed `TopicSpec::partitions() -> Option<i32>` to `TopicSpec::partitions() -> i32` which is a breaking change

Reply:
Got it